extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js

```diff
@@ -451,19 +451,24 @@ class LifecycleBucketProcessor {
     start(done) {
         this.clientManager.initSTSConfig();
         this.clientManager.initCredentialsManager();
-        async.series([
+        const tasks = [
```

Comment:
No need to take this out of the `async.series`, you can just call `done()` when `this._mongoConfig` is not configured.

Reply:
done here : https://github.com/scality/backbeat/pull/2551/commits/f910139f25a006237f08a45c711dc7d4b087fe5b#diff-0251eb0ff96b3559c00aac5d7e592f81df2eece3b644f9278307237dd8f6bbbcR459-R471